src/gooddata.ts

```diff
@@ -58,6 +60,7 @@ export class SDK {
         this.execution = new ExecutionModule(this.xhr, this.md);
         this.project = new ProjectModule(this.xhr);
         this.report = new ReportModule(this.xhr);
+        this.dashboardExport = new DashboardExportModule(this.xhr);
```

Comment:
hmm this is not what I expected...

lets imagine how the user will use it.

`sdk.dashboardExport.toPdf(...)` 
`sdk.report.exportResult(...)`

because I though it would be better to desing it in this way:
`sdk.dashboard.exportToPdf(...)`
`sdk.report.exportResult(...)`

does it make sense?

Reply:
so align packages (folders) to match modules... for example:
`/src/dashboard/dashboard.ts`
`/src/report/report.ts`

Reply:
@martinmosko , do you mean this?
`/src/dashboard/export.ts` and
`src/report/export.ts`

Reply:
Fixed